### ISSUE TITLE
Query about UI feature freeze

### ISSUE THREAD

**Issue by hjuntan** [nus-cs2103-AY2324S2/forum#731]
Hi prof, may I ask if the text truncates when the text field is too long is considered a feature flaw, as these are extreme values and the user can resize the window to resolve these issue; we can update it in the UG as known issues and things the user need to be mindful of. Relating to these issues: https://github.com/AY2324S2-CS2103T-T15-2/tp/issues/184, https://github.com/AY2324S2-CS2103T-T15-2/tp/issues/180

Additionally, can I ask that if the issue of the text is cut off when the window size is too small it is also considered as a feature flaw? Relating to the issue :https://github.com/AY2324S2-CS2103T-T15-2/tp/issues/223

**Comment by damithc** [comment]
@hjuntan Yes, sounds like feature flaws. See Q8 of this FAQ too https://nus-cs2103-ay2324s2.github.io/website/schedule/week12/project.html
